Changes to make eog build with updated gnome-desktop version.  

Updating to a newer version of eog should enable dropping the patch
but that requires an updated librsvg, which requires rust, which is
only 64-bit now.  Modifying librsvg to be delivered 64-bit only will
be a big change, so going with this patch for now

--- a/src/eog-thumbnail.c	2020-01-21 10:06:53.895834077 -0800
+++ b/src/eog-thumbnail.c	2020-01-21 10:08:46.089415268 -0800
@@ -123,9 +123,10 @@
 
 	perc = CLAMP (128.0/(MAX (width, height)), 0, 1);
 
-	thumb = gnome_desktop_thumbnail_scale_down_pixbuf (pixbuf,
-							   width*perc,
-							   height*perc);
+	thumb = gdk_pixbuf_scale_simple (pixbuf,
+					 width*perc,
+                                         height*perc,
+                                         GDK_INTERP_HYPER);
 
 	return thumb;
 }
@@ -444,7 +445,7 @@
 		width  = MAX (width  * factor, 1);
 		height = MAX (height * factor, 1);
 
-		result_pixbuf = gnome_desktop_thumbnail_scale_down_pixbuf (thumbnail, width, height);
+		result_pixbuf = gdk_pixbuf_scale_simple (thumbnail, width, height, GDK_INTERP_HYPER);
 
 		return result_pixbuf;
 	}
